greedybear/settings.py

```diff
@@ -15,6 +15,12 @@
 # SECURITY WARNING: don't run with debug turned on in production!
 DEBUG = os.environ.get("DEBUG", False) == "True"
 
+# DB Backup Config
+# Backup Interval in minutes. Default: 1440 min = 1 day.
+DB_BACKUP_FREQUENCY = int(os.environ.get("DB_BACKUP_FREQUENCY", 1440)) or 1440
+# Backup Storage Location and Max Backups. (When Max Backup is reached old backups are removed).
+DB_BACKUP_STORAGE_OPTIONS = {"location": "/opt/backups", "max_backups": 10}
```

Comment:
I think we can have 2 different variables for those, each configurable like the previous one.